elections/uk/lib.py

```diff
@@ -24,38 +26,43 @@ def shorten_post_label(post_label):
     'party_ec_id',
 ]
 
-def get_extra_csv_values(person, election):
-    theyworkforyou_url = ''
+def get_extra_csv_values(person, election, post):
+    gss_code = ''
```

Comment:
Not sure how much we care, but they're not always GSS codes in the UK, and I expect they never are in some other countries.